Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
+    { 
+
+        const int CYPHERLENGTH = 95; // number of characters before it loops to the start
+        const int UNUSABLE = 32; //the first 32 chars of ascii are not printable
+   
+        public static string EncryptCypher(string message, int shift)
+        {
+            return Cypher(message, shift);
+        }
+
+        public static string DecryptCypher(string message, int shift)
+        {
+
+            shift = Math.Abs(CYPHERLENGTH - shift);
+            
+            return EncryptCypher(message, shift);
+        }
+
+        public static string Cypher(string message, int shift)
```

Comment:
This method is not a Cypher, it does the shifting. So ShiftCharacters is how I would call it.

Reply:
Renamed to ShiftCharacters